behat3.4.docker

```diff
@@ -14,6 +14,9 @@ RUN check_php_version ${MIN_PHP_VERSION} && \
         #
         # instaclick/php-webdriver requirements
         php${PHP_VERSION}-curl && \
+        #
+        # JUnit requirements.
```

Comment:
It's not really JUnit. It's just a part of Behat that is outputting test results using the JUnit format.

```suggestion
        # JUnitOutputPrinter requirements.
```